### ISSUE TITLE
As a developer, I want to know when the session is ready to use

### ISSUE THREAD

**Issue by friedger** [blockstack/blockstack-android#5]
If the first method from the sdk is called within onCreate of the activity then the url with blockstack.js might not yet have finished loading and I get e.g. a the following message:

```
chromium: [INFO:CONSOLE(1)] "Uncaught ReferenceError: redirectToSignIn is not defined", source:  (1)
```

**Comment by friedger** [comment]
as a workaround the first method on the session should be delayed by a second or two.

**Comment by friedger** [comment]
fixed by #8
